### PR TITLE
Footer milestone 1

### PR DESCRIPTION
**Pull request by hoangviet1993**
This PR sets out to setup the basic layout for footer. Some attempt was made with regards to responsive design.
Some pic: 
Just the footer
screen shot 2018-08-06 at 11 39 03 pm

Desktop:
screencapture-localhost-8181-2018-08-06-23_39_12
Mobile:
screencapture-localhost-8181-2018-08-06-23_39_48

### CODE REVIEW

**Comment by hoangviet1993** [comment]
PTAL @seanlip

File: pages/footer/footer.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    <md-content layout="column" layout-align="center center" layout-align-gt-sm="start" class="footer-background-color oppia-foundation-horizontal-padding">
+      <h4 class="oppia-foundation-uppercase-text oppia-foundation-white-text footer-title">Contact us</h4>
+      <p class="footer"><a>info@oppiafoundation.org</a></p>
+    </md-content>
+    <md-content layout="column" layout-align="center center" layout-align-gt-sm="start center" class="footer-background-color oppia-foundation-horizontal-padding">
+      <h4 class="oppia-foundation-uppercase-text oppia-foundation-white-text footer-title">Follow us</h4>
+      <md-content  layout="row" layout-align="center center" layout-gt-md="row" class="footer-background-color">
+        <md-button class="social-media-button google-plus-button" md-no-ink>
+          <md-icon md-svg-src="/static/icons/google-plus-g-brands.svg"
+                  aria-label="G+ icon"
```
Comment (seanlip) [inline pages/footer/footer.html]: Here and below: typically we indent this by one more space to match the attr in the row above it.

File: pages/footer/footer.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      <p class="footer"><a>Browse Topics</a></p>
+      <p class="footer"><a>Contribute to Our Lessons</a></p>
+    </md-content>
+    <md-content layout="column" layout-align="center center" layout-align-gt-sm="start" class="footer-background-color oppia-foundation-horizontal-padding">
+      <h4 class="oppia-foundation-uppercase-text oppia-foundation-white-text footer-title">Contact us</h4>
+      <p class="footer"><a>info@oppiafoundation.org</a></p>
+    </md-content>
+    <md-content layout="column" layout-align="center center" layout-align-gt-sm="start center" class="footer-background-color oppia-foundation-horizontal-padding">
+      <h4 class="oppia-foundation-uppercase-text oppia-foundation-white-text footer-title">Follow us</h4>
+      <md-content  layout="row" layout-align="center center" layout-gt-md="row" class="footer-background-color">
```
Comment (seanlip) [inline pages/footer/footer.html]: Remove duplicate space before "layout"

Comment (hoangviet1993) [reply pages/footer/footer.html]: Thanks for catching this!

File: pages/footer/footer.html (hunk 1 above)
Comment (hoangviet1993) [reply pages/footer/footer.html]: I was going by tab spaces and wasn't sure what do do here. Thanks for the pointer :)

**Comment by hoangviet1993** [comment]
Thanks and merging! 👍

**Pull request merged**